Chummer/Character Creation/frmCreate.cs

```diff
@@ -4993,12 +4993,13 @@ private void tsVehicleGearNotes_Click(object sender, EventArgs e)
             {
                 case Gear objGear:
                 {
-                    using (frmNotes frmItemNotes = new frmNotes(objGear.Notes))
+                    using (frmNotes frmItemNotes = new frmNotes(objGear.Notes, objGear.NotesColor))
```

Comment:
I didn't dare to change, but maybe it would be even more valuable to have a Notes class which can hold several things per object.
For now that would be
* the text
* the ForegroundColor

and might be extended to
* an icon
* the background color